packages/quantic/force-app/main/default/lwc/quanticResultList/e2e/pageObject.ts

```diff
@@ -0,0 +1,90 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaClickEvent} from '../../../../../../playwright/utils/requests';
+
+export class ResultListObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get performSearchButton(): Locator {
+    return this.page.locator('c-action-perform-search button');
+  }
```

Comment:
This is already available with the search object, no need to redefine it: https://github.com/coveo/ui-kit/blob/dc880bbfb8b1d3160be733236aa5543708fdcbd9/packages/quantic/playwright/page-object/searchObject.ts#L12

Reply:
oh really? I've been adding this everywhere without checking 🙈